student/src/components/Pages/Home/index.style.js

```diff
@@ -1,64 +1,62 @@
 import styled from "styled-components";
+import Popup from 'reactjs-popup';
 
```

Comment:
The line 2 needs to be removed, its causing error since `reactjs-popup` is uninstalled and no longer used